tests/gem5/cpu_tests/test.py

```diff
@@ -49,7 +49,8 @@
     for workload in workloads:
         ref_path = joinpath(getcwd(), 'ref', workload)
         verifiers = (
-                verifier.MatchStdout(ref_path),
+                verifier.MacthStdout(ref_path),
+                verifier.MatchStats(ref_path),
```

Comment:
Two questions:
1) doesn't MatchStats take in only the list of stats?
2) Shouldn't MatchStats also need the ref_path?